unlock-protocol-com/blog/checking-key-in.md

```diff
@@ -1,36 +1,38 @@
 
-<p style="text-align:center" ><img width="300" src="/images/blog/checking-key-in/keychain.png" alt="key in keychain"></p>
+<p style="text-align:center" ><img width="300" src="/images/blog/checking-key-in/devcon-demo-ticket.jpg" alt="key in keychain"></p>
 
-With their signature we generate a QR code which can then be used by anyone to verify that indeed, this user owns a key and that this key is valid. In fact, the QR code includes a full URL can be loaded to verify if the key is valid. Below is my key to EthCC. You can scan it with your camera application!
+For each ticket, we generate a QR code which can then be used by anyone to verify that indeed, this user owns this ticket and that this ticket is valid. Below is my QR code for a ticket. You can scan it with your camera application!
 
-<p style="text-align:center" ><img width="300" src="/images/blog/checking-key-in/ethcc-ticket-qrcode.png" alt="qr code ticket"></p>
+<p style="text-align:center" ><img width="300" src="/images/blog/checking-key-in/qr-code.png" alt="qr code ticket"></p>
 
 # Checking users in
 
-Unlock keys are, by default, transferable. This means that, unless we use a mechanism to prevent this, someone could use their key (and the corresponding QR code) to enter a venue, transfer the key to another user, who would re-use the same key (and another version of the QR code) to enter the venue as well. **This is not how tickets are supposed to work**!
+In a small number of cases, some individuals may try to transfer their ticket to someone after they have used it. This means that, unless we use a mechanism to prevent this, an unscrupulous individual could use their ticket (and the corresponding QR code) to enter a venue, and then transfer the ticket to another user who is waiting outside, who would re-use the same ticket (and another version of the QR code) to enter the venue as well. **This is not how tickets are supposed to work**!
 
-The verification page above works from any web browser... but, if viewed from inside a browser which has a crypto wallet and if the viewer is actually the lock owner, **the viewer can "mark" the ticket as checked-in**. For this, we are using metadata on the NFT which can only be set by the lock owner. This way, if the ticket is then transfered to someone else, the organizers would still be able to quickly identify sneaky users who are trying to re-use a ticket.
+Unlock has built-in mechanisms to prevent this, by chacking in tickets at the venue and marking those tickets as "used" for event organizers who wish to do so. The verification page above works from any web browser. For this, we are using metadata on the NFT which can only be set by the event organizer (lock owner). This way, if the ticket is then transfered to someone else, the organizers would still be able to quickly identify sneaky users who are trying to re-use a ticket.
```

Comment:
```suggestion
Unlock has built-in mechanisms to prevent this, by checking in tickets at the venue and marking those tickets as "used" for event organizers who wish to do so. The verification page above works from any web browser. For this, we are using metadata on the NFT which can only be set by the event organizer (lock owner). This way, if the ticket is then transfered to someone else, the organizers would still be able to quickly identify sneaky users who are trying to re-use a ticket.
```